lib/disposable.dart

```diff
@@ -192,29 +250,38 @@ class Disposable implements _Disposable, DisposableManager {
         .then(_completeDisposeFuture);
   }
 
-  /// Automatically dispose another object when this object is disposed.
-  ///
-  /// The parameter may not be `null`.
+  /// Creates a [Timer] instance that will be cancelled if active upon disposal.
+  @mustCallSuper
+  @override
+  Timer getManagedTimer(Duration duration, void callback()) {
+    var timer = new _ObservableTimer(duration, callback);
+    _addObservableTimerDisposable(timer);
+    return timer;
+  }
+
+  /// Creates a periodic [Timer] that will be cancelled if active upon disposal.
```

Comment:
You probably don't want doc strings here since they're on the `DisposableManager` abstract class, right?